CleverPush/Source/CPStoriesController.m

```diff
@@ -124,6 +124,9 @@ - (UIView *)carousel:(CleverPushiCarousel *)carousel viewForItemAtIndex:(NSInteg
     webview.backgroundColor = [UIColor clearColor];
     webview.scrollView.backgroundColor = [UIColor clearColor];
     webview.opaque = false;
+    webview.urlOpenedListener = ^(NSURL *url) {
+        NSLog(@"Tracked Url = %@",url);
```

Comment:
Remove this log
The customer's callback should be called here instead. Customer can set his own `urlOpenedListener`. Currently this is not possible.

Reply:
So right now, We just need to remove this line?

```
webview.urlOpenedListener = ^(NSURL *url) {
        NSLog(@"Tracked Url = %@",url);
    };
```
[](url)
The functionality will work like this
If the customer sets the above function, they will get the return URL or callback, and if the customer does not set the above function, then the old functionality will work the same as it did before. Let me share the video.